.eslintrc.json

```diff
@@ -56,6 +58,18 @@
     ],
     "quotes": ["error", "single", { "allowTemplateLiterals": true, "avoidEscape": true }]
   },
+  "settings": {
+    "import/core-modules": 
+      [
+        "googlesitekit-data",
+        "googlesitekit-modules",
+        "googlesitekit-api",
+        "googlesitekit-widgets",
+        "@wordpress/i18n__non-shim",
+        "@wordpress/api-fetch__non-shim",
```

Comment:
Is this ok? How come they aren't dependencies in package.json like the other @wordpress packages :thinking: 

I can look at what these are next

Reply:
@aaemnnosttv what are these modules? They 404 on NPM :thinking:

Reply:
These are shims we need to use to configure the WordPress API and I18n packages: https://github.com/google/site-kit-wp/blob/1757a849eae1e77e13a2a092eaaa645d3c14dff8/webpack.config.js#L120-L122

They're named this way, with aliases, I suppose to make it clear what they're referencing/working with, but I suppose that makes it a bit unclear too 😞 

Anyway, they'd also be great to reference in the ESLint config as the paths specified in the Webpack config like the `googlesitekit-*` JS files, but otherwise ignoring them is fine.